testrunner-running/src/main/scala/edu/illinois/cs/testrunner/execution/Executor.scala

```diff
@@ -29,6 +29,9 @@ object Executor {
             case "JUnit" =>
                 JUnitTestExecutor.runOrder(testRunId, tests, true, false)
                     .writeTo(outputPath.toAbsolutePath.toString)
+            case "JUnit5" =>
+                JUnit5TestExecutor.runTestsSeparately(testRunId, tests)
+                                  .writeTo(outputPath.toAbsolutePath.toString)
```

Comment:
Shift the whitespace here to match how it is for line 31.

Reply:
I think line 34 looks better than line 31. How about I change line 31 to match line 34?

Reply:
That's fine, we do want consistency here. We should eventually implement some checkstyle for this codebase as well.

Reply:
Done

Reply:
It looks like you are off by one space still on either line 31 or 34.

Reply:
I think the rule is to match the dot position with the above line.